controlplane/kubeadm/internal/workload_cluster_conditions.go

```diff
@@ -234,7 +350,7 @@ func compareMachinesAndMembers(controlPlane *ControlPlane, members []*etcd.Membe
 			if name == "" {
 				name = fmt.Sprintf("%d (Name not yet assigned)", member.ID)
 			}
-			kcpErrors = append(kcpErrors, fmt.Sprintf("etcd member %s does not have a corresponding machine", name))
+			kcpErrors = append(kcpErrors, fmt.Sprintf("Etcd member %s does not have a corresponding machine", name))
```

Comment:
There are a few more lower case etcd at the beginning of strings.

Reply:
fixed them all, also for older conditions